R/submit-models.R

```diff
@@ -0,0 +1,153 @@
+
```

Comment:
I feel the same way about this now. I'm gonna be glad we did it, but it does make the diff a little uglier.